### DISCUSSION TITLE
Arenas

### DISCUSSION THREAD

**Discussion by fanat1cc** [diamond-rip/Eden#501] [Q&A]
How do I create an arena. Do I need another plugin? I have a sumo kit but I have no idea how to create an arena. Would be nice if in the wiki there was a full explanation on how to do this
I understand fwae is involved but can someone explain exactly how I would go about creating an arena.
I have sumo maps btw

**Comment by GoodestEnglish** [comment]
There is wiki. It is in here https://github.com/diamond-rip/Eden/wiki/Arenas-Setup
